src/main/kotlin/graphics/scenery/volumes/Volume.kt

```diff
@@ -209,6 +209,15 @@ open class Volume(
             field = value
         }
 
+    val bytesPerVoxel: Int
+        get() {
+            return when(dataSource){
+                VolumeDataSource.NullSource -> 1
+                is VolumeDataSource.RAISource<*> -> dataSource.type.toBytesPerValue()
+                is SpimDataMinimalSource -> (dataSource.sources.first().spimSource.type as NumericType<*>).toBytesPerValue() //I dont know if this works
```

Comment:
If you don't know if this works, please test it. Let's not have some untested code here that might bite down the line 👍

Reply:
I dont know how SpimDataMinimalSource work and it is not documented.

Reply:
This is used every time you load a BDV format (XML/HDF5) dataset, you can therefore test it with any of these.